CHANGELOG.md

```diff
@@ -1,14 +1,43 @@
 # Parse-SDK-iOS-OSX Changelog
 
 ### master
-[Full Changelog](https://github.com/parse-community/Parse-SDK-iOS-OSX/compare/1.17.3...master)
+[Full Changelog](https://github.com/parse-community/Parse-SDK-iOS-OSX/compare/1.18.0...master)
 * _Contributing to this repo? Add info about your change here to be included in next release_
 
+### 1.18.0
+[Full Changelog](https://github.com/parse-community/Parse-SDK-iOS-OSX/compare/1.17.3...1.18.0)
+
+__New features__
+- Add get and set server functions ([#1464](https://github.com/parse-community/Parse-SDK-iOS-OSX/pull/1464)), thanks to [Manuel Trezza](https://github.com/mtrezza).
+
+__Improvements__
+- Swift 5 compatibility, thanks to [Nathan Kellert](https://github.com/noobs2ninjas).
+- macOS 10.15 compatibility thanks to [Martin Man](https://github.com/mman).
```

Comment:
Can we get some PR links in here and below? - is this macOS one referring to #1460 and/or #1469?

Reply:
I made a commit with the PRs linked.